### PR TITLE
Covid19 Postmortem Report - WIP

### PR DESCRIPTION
**Pull request by ODORA0**
https://app.asana.com/0/inbox/1180757605022193/1200346445396459/1200347509513809

### CODE REVIEW

**Comment by semujju** [comment]
- Rename PR to Covid19 Postmortem Report - WIP
- Use sentence case in Postmortem every where on the form
- Balance the calendar alignment in date boxes

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml [hunk 1]
```diff
@@ -0,0 +1,248 @@
+<htmlform formName="Covid-19 PostMortem Report"
```
Comment (semujju) [inline omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml]: Use sentence case for PostMortem

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+          formEncounterType="422ee220-9e83-451d-9b25-79a688a0413a"
+          formUuid="9644a450-5027-4584-8e74-e64a4c7cd347" 
+          formVersion="3.0.0"
+          formAddMetadata="yes"
+          formUILocation="patientDashboard.visitActions"
+          formOrder="1"
+          formIcon="icon-medkit"
+          formShowIf="(visit.active || !visit.active) &amp;&amp; patient.person.dead==false&amp;&amp; sessionLocation.uuid=='1748bd6f-b013-449f-8d38-91319b576f3f'"
+          formDisplayStyle="Standard"
+          formLabel="Covid-19 PostMortem Report">
```
Comment (semujju) [inline omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml]: Use sentence case for PostMortem

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+
+
+    <div>
+        <div class="row">
+            <div class="col-md-6">
+                <h1>Covid-19: PostMortem Report</h1>
```
Comment (semujju) [inline omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml]: Use sentence case for PostMortem

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col-md-12">
+                <div class="card">
+                    <div class="card-header">
+                        <ul class="nav nav-tabs nav-fill card-header-tabs">
+                            <li class="nav-item">
+                                <a class="nav-link active" data-toggle="tab" href="#patient-info-section">Post Mortem Report</a>
```
Comment (semujju) [inline omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml]: Rename to Patient Information

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml [hunk 5]
```diff
@@ -0,0 +1,248 @@
...
+												<i class="icon-calendar small right-addon-row position-absolute" style="right: 25%; top: 47%;"></i>
+												<obs conceptId="90272" />
+											</div>
+                                            <div class="col-md-3">
+                                                <label>Covid symptoms</label>
+                                                <obs conceptId="159395" style="textarea" />
+                                            </div>
+                                            <div class="col-md-3">
+												<label>Date of Reciept of Body:</label>
+												<i class="icon-calendar small right-addon-row position-absolute" style="right: 25%; top: 47%;"></i>
```
Comment (semujju) [inline omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml]: Make sure calenders is not flying over

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml [hunk 6]
```diff
@@ -0,0 +1,248 @@
...
+												<obs conceptId="159376" />
+                                            </div>
+                                            <div class="col-md-3">
+												<label>Date of Admission in Hospital:</label>
+												<i class="icon-calendar small right-addon-row position-absolute" style="right: 25%; top: 47%;"></i>
```
Comment (semujju) [inline omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml]: Make sure calenders is not flying over

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml [hunk 7]
```diff
@@ -0,0 +1,248 @@
...
+                                                     style="radio" class="horizontal"/>
+                                            </div>
+                                            <div class="col-md-3">
+                                                <label>How was the diagnosis confirmed</label>
+                                                <obs id="1542" conceptId="1542"
+                                                     answerConceptIds="166272,159466,166273,5622"
+                                                     answerLabels="Ante mortem Lab Diagnosis, Ante mortem Radiological Diagnosis, Ante mortem Clinical Diagnosis, Post-Mortem Lab Diagnosis"/>
+                                            </div>
+                                            <div class="col-md-3">
+												<label>If yes, indicate Date of Diagnosis:</label>
```
Comment (semujju) [inline omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml]: Make sure calenders is not flying over

Comment (semujju) [reply omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml]: - Use Date of Diagnosis as a label
- Deactivate by default
- Activate if yes

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml [hunk 8]
```diff
@@ -0,0 +1,248 @@
...
+												<i class="icon-calendar small right-addon-row position-absolute" style="right: 25%; top: 47%;"></i>
+												<obs conceptId="90272" />
+											</div>
+                                            <div class="col-md-3">
+                                                <label>Diagnosis Type</label>
+                                                <obs id="166285" conceptId="166285" answerConceptIds="90003,90004" answerLabels="ANTEMORTEM CASE UNDER INVESTIGATION, POST-MORTEM EXAMINATION (GROSS) DIAGNOSIS"
```
Comment (semujju) [inline omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml]: - Use lower case for answers
- Move it (Diagnosis type) to second row

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml [hunk 9]
```diff
@@ -0,0 +1,248 @@
...
+                                        <obs id="1542" conceptId="1542"
+                                             answerConceptIds="166272,159466,166273,5622"
+                                             answerLabels="cannula, I.V line, NGT, catheters, Other Specify"/>
+                                    </div>
+                                    <div class="col-md-4">
+                                        <label>Other Medical Intervention</label>
+                                        <obs id="165136" conceptId="165136"/>
+                                    </div>
+                                    </div>
+                                    <div class="row">
```
Comment (semujju) [inline omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml:215]: - Put four checkbox options per row
- Put other (checkbox) and Specify (text) on a separate row

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml [hunk 10]
```diff
@@ -0,0 +1,248 @@
...
+                                    <div class="col-md-4">
+                                        <label>Cause of Death</label>
+                                        <obs id="5002" conceptId="5002"
+                                             answerConceptIds="90001,99288,99287,58,165374,165375,165376,165377,116030"
+                                             answerLabels="UNKNOWN, DISEASE, ACCIDENT, TUBERCULOSIS, Infectious Disease other than TB, HIV Non-infectious Disease, Natural Causes, Non natural causes, Cancer"/>
+                                    </div>
+                                    <div class="col-md-4">
+                                        <label>Medical Officer</label>
```
Comment (semujju) [inline omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml]: Use provider tag to provide a drop of all provider

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml [hunk 11]
```diff
@@ -0,0 +1,248 @@
...
+                                    </div>                                 
+                                </div>
+                            </div>
+
+                            <div class="card">
+                                <div class="card-header">
+                                    Internal Examination
+                                </div>
+                                <div class="card-body">
+                                <div class="row">
```
Comment (semujju) [inline omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml:245]: Use a mb to create space between rows

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml (hunk 1 above)
Comment (ODORA0) [reply omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml]: @semujju Done

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml (hunk 2 above)
Comment (ODORA0) [reply omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml]: @semujju Done

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml (hunk 3 above)
Comment (ODORA0) [reply omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml]: @semujju Done

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml (hunk 4 above)
Comment (ODORA0) [reply omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml]: @semujju Done

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml (hunk 5 above)
Comment (ODORA0) [reply omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml]: @semujju Done

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml (hunk 6 above)
Comment (ODORA0) [reply omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml]: @semujju Done

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml (hunk 7 above)
Comment (ODORA0) [reply omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml]: @semujju Done

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml (hunk 8 above)
Comment (ODORA0) [reply omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml]: @semujju Done

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml (hunk 9 above)
Comment (ODORA0) [reply omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml:215]: @semujju Done

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml (hunk 10 above)
Comment (ODORA0) [reply omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml]: @semujju Done

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml (hunk 11 above)
Comment (ODORA0) [reply omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml:245]: @semujju Done

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml [hunk 12]
```diff
@@ -0,0 +1,341 @@
...
+                                    </div>
+                                </div>
+
+                                <div class="card">
+                                    <div class="card-header">
+                                        Diagnosis History
+                                    </div>
+                                    <div class="card-body">
+                                        <div class="row">
+                                            <div class="col-md-3">
```
Comment (chosendev) [inline omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml:153]: Use col-md-4

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml [hunk 13]
```diff
@@ -0,0 +1,341 @@
...
+                                                <label>Covid Confirmed</label>
+                                                <obs id="covid-confirmed" conceptId="166285" answerConceptIds="90003,90004" answerLabels="Yes, No"
+                                                     style="radio" class="horizontal"/>
+                                            </div>
+                                            <div class="col-md-9">
```
Comment (chosendev) [inline omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml]: use col-md-8

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml [hunk 14]
```diff
@@ -0,0 +1,341 @@
...
+                                                <label>Diagnosis Type</label>
+                                                <obs id="diagnosis-type" conceptId="166285" answerConceptIds="90003,90004" answerLabels="Antemortem case under investigation, Post-mortem Examination (gross) diagnosis"
+                                                     style="radio" class="horizontal"/>
+                                            </div>
+                                            </div>
+                                            <div class="row">
+                                            <div class="col-md-4">
+                                                <label>How was the diagnosis confirmed</label>
```
Comment (chosendev) [inline omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml]: Put a question mark at the end

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml [hunk 15]
```diff
@@ -0,0 +1,341 @@
...
+                                                <obs id="1542" conceptId="1542"
+                                                     answerConceptIds="166272,159466,166273,5622"
+                                                     answerLabels="Ante mortem Lab Diagnosis, Ante mortem Radiological Diagnosis, Ante mortem Clinical Diagnosis, Post-Mortem Lab Diagnosis"/>
+                                            </div>
+                                            <div class="col-md-3">
+												<label>Date of Diagnosis</label>
```
Comment (chosendev) [inline omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml:159]: Use proper indention

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml [hunk 16]
```diff
@@ -0,0 +1,341 @@
...
+                                            <label>Specify</label>
+                                            <obs id="specify" conceptId="166296" />
+                                        </div>
+                                    </div>
+                                    <div class="row">
+                                    <div class="col-md-4">
+                                        <label>Cause of Death</label>
+                                        <obs id="5002" conceptId="5002"
+                                             answerConceptIds="90001,99288,99287,58,165374,165375,165376,165377,116030"
+                                             answerLabels="UNKNOWN, DISEASE, ACCIDENT, TUBERCULOSIS, Infectious Disease other than TB, HIV Non-infectious Disease, Natural Causes, Non natural causes, Cancer"/>
```
Comment (chosendev) [inline omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml]: Use Sentence case for all answers

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml [hunk 17]
```diff
@@ -0,0 +1,341 @@
...
+
+                jq(".166319").change(function () {
+                    if (jq(this).find('#other').find(':checkbox').prop('checked')) {
+                        enable_fields("specify");
+                    } else {
+                        disable_fields("specify"); 
+                    }
+                });
+
+                jq("#covid-confirmed").click(function () {
```
Comment (chosendev) [inline omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml:37]: This should enable "Diagnosis Type" first, the "Diagnosis Type" should enable "How was the diagnosis confirmed" then "How was the diagnosis confirmed" should enable "Date of Diagnosis"

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml [hunk 18]
```diff
@@ -0,0 +1,341 @@
+<htmlform formName="Covid-19 Postmortem Report"
```
Comment (chosendev) [inline omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml]: Use Covid-19 Postmortem

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml (hunk 12 above)
Comment (ODORA0) [reply omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml:153]: @semujju Done

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml (hunk 18 above)
Comment (ODORA0) [reply omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml]: @semujju Done

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml (hunk 16 above)
Comment (ODORA0) [reply omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml]: @semujju Done

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml (hunk 15 above)
Comment (ODORA0) [reply omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml:159]: @semujju Github renders it this way but on my IDE prettier renders it well

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml (hunk 14 above)
Comment (ODORA0) [reply omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml]: @semujju Done

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml (hunk 13 above)
Comment (ODORA0) [reply omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml]: @semujju Done

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml (hunk 17 above)
Comment (ODORA0) [reply omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml:37]: @semujju This was fixed accordingly

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml [hunk 19]
```diff
@@ -0,0 +1,344 @@
...
+                                                <label>Covid Confirmed</label>
+                                                <obs id="covid-confirmed" conceptId="166285" answerConceptIds="90003,90004" answerLabels="Yes, No"
+                                                     style="radio" class="horizontal"/>
+                                            </div>
+                                            <div class="col-md-3">
+												<label>Date of Diagnosis</label>
+												<i class="icon-calendar small right-addon-row position-absolute" style="right: 25%; top: 45%;"></i>
+												<obs id="date-of-diagnosis" conceptId="90272" />
+											</div>
+                                            <div class="col-md-4">
```
Comment (semujju) [inline omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml]: use col-md-3

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml [hunk 20]
```diff
@@ -0,0 +1,344 @@
...
+                                    </div>
+                                </div>
+
+                                <div class="card">
+                                    <div class="card-header">
+                                        Diagnosis History
+                                    </div>
+                                    <div class="card-body">
+                                        <div class="row">
+                                            <div class="col-md-4">
```
Comment (semujju) [inline omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml]: use col-md-3

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml (hunk 19 above)
Comment (ODORA0) [reply omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml]: @semujju Done

File: omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml (hunk 20 above)
Comment (ODORA0) [reply omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml]: @semujju Done

**Pull request merged**